test/unit/hooks/sdtdlogs/LoggingObject.test.js

```diff
@@ -1,4 +1,5 @@
 const LoggingObject = require('../../../../api/hooks/sdtdLogs/LoggingObject');
+const { expect } = require('chai');
```

Comment:
```suggestion
```

its a global import you shouldn't need it.

Reply:
Ah man my editor does that automatically >.<